package.json

```diff
@@ -50,11 +50,11 @@
     "globalize": "1.4.0",
     "immutable": "3.8.2",
     "intersection-observer": "0.4.2",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
`cross-fetch` might be a better option for this (I haven't used either): https://github.com/lquixada/cross-fetch#why-not-isomorphic-fetch

Reply:
Good point, I've updated this to use `cross-fetch`